src/components/Select/Select.test.tsx

```diff
@@ -53,7 +53,7 @@ describe('<SelectMulti />', () => {
     const label = 'MultiLabel';
     const maxSelections = 2;
     const user = userEvent.setup();
-    const onChange = jest.fn();
+    const onChange = fn();
```

Comment:
Replacing these with the `@storybook/test` method of mocking out these functions.